stable/airflow/values.yaml

```diff
@@ -128,6 +128,13 @@ airflow:
   ##     type: aws
   ##     extra: '{"aws_access_key_id": "**********", "aws_secret_access_key": "***", "region_name":"eu-central-1"}'
   connections: {}
+  ##
+  ## Annotations for the Web and Scheduler Pods
+  pod:
```

Comment:
Please look at Kibana Chart as a source of inspiration
https://github.com/helm/charts/blob/master/stable/kibana/templates/deployment.yaml